.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = tab
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{package.json,package-lock.json}]
+indent_size = 2
+indent_style = space
```

Comment:
Let's not use this please. This lib is not intended be opened in the editor.

Reply:
I'm not sure I understand.. How do you edit the source code if not in an editor?

Reply:
Removed.

Reply:
Thanks!